sys/include/luid.h

```diff
@@ -28,20 +28,16 @@
  * value with the LSB of the base ID, or (ii) by XORing the least significant
  * byes with a value given by the user.
  *
- * Example: Calling `uuid_base(&buf, 8)` will always yield an identical  value,
+ * Example: Calling `luid_base(&buf, 8)` will always yield an identical  value,
  * independent how often the function is called. But calling
- * `uuid_base(&buf, 2)` afterwards will results in a different value, if the
+ * `luid_base(&buf, 2)` afterwards will results in a different value, if the
  * cpuid module is present, and in the same (but shorter) value if not.
  *
- * Example: Calling `uuid_get(&buf, 8)` four times in a row, will yield four
+ * Example: Calling `luid_get(&buf, 8)` four times in a row, will yield four
  * different IDs, differing in their LSB.
  *
- * Example: Calling `uuid_custom(&buf, 8, 123)` will always yield the same
- * value, but calling `uuid_custom(&buf, 8, 124)` will differ.
- *
- * @note        This module generates unique IDs without any guarantees on their
- *              structure. These UUIDs are not compatible nor conform to the
- *              UUIDs as defined in RFC4122.
```

Comment:
I think you could leave this comment intact, even under a new module name. It's good to be clear in the api documentation to avoid any possible confusion.